ferrocene/tools/self-test/src/error.rs

```diff
@@ -17,12 +17,13 @@ pub(crate) enum LinkerArgsErrorKind {
 }
 
 #[derive(Debug)]
+#[allow(clippy::enum_variant_names)]
```

Comment:
what do you think about renaming LinkerArgsError to just LinkerArgs, to avoid this allow